boards/adi/max32657evkit/doc/index.rst

```diff
@@ -115,196 +115,203 @@ Connections and IOs
 +-----------+---------------+---------------+--------------------------------------------------------------------------------------------------+
 | Name      | Name          | Settings      | Description                                                                                      |
 +===========+===============+===============+==================================================================================================+
-| JP1       | VDD12 SEL     |               |                                                                                                  |
-|           |               | +-----------+ |  +-------------------------------------------------------------------------------+               |
-|           |               | | 1-2       | |  | VDD12 User Selection.                                                         |               |
-|           |               | +-----------+ |  +-------------------------------------------------------------------------------+               |
-|           |               | | 3-4       | |  |  VDD12 AT 1.2 VOLTS.                                                          |               |
-|           |               | +-----------+ |  +-------------------------------------------------------------------------------+               |
-|           |               | | 5-6       | |  | VDD12 AT 1.5 VOLTS.                                                           |               |
-|           |               | +-----------+ |  +-------------------------------------------------------------------------------+               |
-|           |               | | Open      | |  |  VDD12 AT 1.6 VOLTS.                                                          |               |
```

Comment:
fix documentation in original commit

Reply:
Pushed as a separate commit to give credit to committer.

Reply:
Agree with @nordicjm . Squash into the original commit and add @hfakkiz 's signed-off-by line

Reply:
Done.